extension/js/agent/utils/serializeObject.js

```diff
@@ -90,13 +90,28 @@
       return this.classPropertyCache[info.name];
     }
 
-    var props = this.serializeObject(_.pick(serializeObject, info.keys));
+    // :FIXME: Properties should always be resolved on `object`.  This might
+    // break memoization though.
```

Comment:
these are actually different (serializeObject[key] / object[key]) if the value is overrode.